### DISCUSSION TITLE
Question on pytest_keyboard_interrupt

### DISCUSSION THREAD

**Discussion by eleanorjboyd** [pytest-dev/pytest#11199] [Q&A]
Quick question on pytest_keyboard_interrupt. When I use a pytest plugin and the tests I am running my plugin on have a ModuleNotFoundError my plugin calls both the pytest_exception_interact hook and then the pytest_keyboard_interrupt hook. Why is pytest_keyboard_interrupt called for a ModuleNotFoundError? Is there any way to make it so it hits only one of the exception hooks? I am trying to keep track of the number of errors hit during discovery and a single error triggering two error hooks is making that more complicated. Also open to suggestions regarding another work-around. Thanks!

**Comment by The-Compiler** [comment]
This seems like a bug - I thus opened #11200.

**Comment by eleanorjboyd** [comment]
Great thank you! Ill close this since we now have an issue to track it.
